pydoof/helpers.py

```diff
@@ -28,7 +29,7 @@ def parse_query_params(params):
     return query_params
 
 
-def _parse_param(param, value):
+def _parse_param(param: str, value: Any, counter: int = 0):
```

Comment:
`counter`? I don't get it...

Reply:
This is because of recursion, It's needed to allow `i` from `enumerate`. Yes, isn't the best name, I could replaced it for `facet_number`, `facet_index` or something like that. I accepted suggestions @mdogo.

Reply:
No, I think you got me wrong. What I meant is that you are not using `counter`.

Reply:
Just run the test and you will see this error if you remove counter: `TypeError: _parse_param() takes 2 positional arguments but 3 were given`

Reply:
Yes, because you are given it a parameter that you are not using below.

https://github.com/doofinder/pydoof/blob/0d0236789651e1300c13e6b179fffefec568d030/pydoof/helpers.py#L46-L48

Reply:
Yes, thanks, changed.